docker-compose.yaml

```diff
@@ -36,18 +23,8 @@ services:
             MYSQL_PASSWORD: "secret"
             MYSQL_ROOT_PASSWORD: "secret"
         ports:
-            - "3306:3306"
+            - "3307:3306"
```

Comment:
This was conflicting with the local database, so we adjusted it to a non-default MySQL port.